[2024-03-21T15:26:13.114-0300] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: executing_multiple_tasks2.task_D manual__2024-03-21T18:26:00.606341+00:00 [queued]>
[2024-03-21T15:26:13.118-0300] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: executing_multiple_tasks2.task_D manual__2024-03-21T18:26:00.606341+00:00 [queued]>
[2024-03-21T15:26:13.119-0300] {taskinstance.py:1288} INFO - 
--------------------------------------------------------------------------------
[2024-03-21T15:26:13.119-0300] {taskinstance.py:1289} INFO - Starting attempt 1 of 1
[2024-03-21T15:26:13.119-0300] {taskinstance.py:1290} INFO - 
--------------------------------------------------------------------------------
[2024-03-21T15:26:13.130-0300] {taskinstance.py:1309} INFO - Executing <Task(BashOperator): task_D> on 2024-03-21 18:26:00.606341+00:00
[2024-03-21T15:26:13.131-0300] {standard_task_runner.py:55} INFO - Started process 23696 to run task
[2024-03-21T15:26:13.133-0300] {standard_task_runner.py:82} INFO - Running: ['airflow', 'tasks', 'run', 'executing_multiple_tasks2', 'task_D', 'manual__2024-03-21T18:26:00.606341+00:00', '--job-id', '28', '--raw', '--subdir', 'DAGS_FOLDER/multiple_tasks2.py', '--cfg-path', '/tmp/tmp82ihk54e']
[2024-03-21T15:26:13.134-0300] {standard_task_runner.py:83} INFO - Job 28: Subtask task_D
[2024-03-21T15:26:13.167-0300] {task_command.py:389} INFO - Running <TaskInstance: executing_multiple_tasks2.task_D manual__2024-03-21T18:26:00.606341+00:00 [running]> on host DESKTOP-0DNP0LN.localdomain
[2024-03-21T15:26:13.209-0300] {taskinstance.py:1518} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=falcone
AIRFLOW_CTX_DAG_ID=executing_multiple_tasks2
AIRFLOW_CTX_TASK_ID=task_D
AIRFLOW_CTX_EXECUTION_DATE=2024-03-21T18:26:00.606341+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-03-21T18:26:00.606341+00:00
[2024-03-21T15:26:13.209-0300] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2024-03-21T15:26:13.210-0300] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo Task D has completed!']
[2024-03-21T15:26:13.213-0300] {subprocess.py:86} INFO - Output:
[2024-03-21T15:26:13.213-0300] {subprocess.py:93} INFO - Task D has completed!
[2024-03-21T15:26:13.214-0300] {subprocess.py:97} INFO - Command exited with return code 0
[2024-03-21T15:26:13.228-0300] {taskinstance.py:1332} INFO - Marking task as SUCCESS. dag_id=executing_multiple_tasks2, task_id=task_D, execution_date=20240321T182600, start_date=20240321T182613, end_date=20240321T182613
[2024-03-21T15:26:13.265-0300] {local_task_job.py:212} INFO - Task exited with return code 0
[2024-03-21T15:26:13.273-0300] {taskinstance.py:2596} INFO - 0 downstream tasks scheduled from follow-on schedule check
